File: 1512. Number of Good Pairs.ts
function numIdenticalPairs(nums: number[]): number {
    let res = 0;

    for(let i = 0; i < nums.length; i++){
        for(let j = i+1; j <nums.length; j++){
            if(nums[i] == nums[j]){
                res++;
            }
        }
    }

    return res;

File: 1512. Number of Good Pairs.py
class Solution(object):
    def numIdenticalPairs(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = 0
        map = {}

        for num in nums:
            if num in map:
                count += map[num]
                map[num] += 1
            else:
                map[num] = 1

        return count
